HANDLING-REPORTS.md

```diff
@@ -0,0 +1,37 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+Reporting to the OpenJS Foundation either concerns a matter with the Foundation or is considered an escalation where insufficient resolution was provided by a project.
+
+## Channels
+* Email: report@openjsf.org
+  * This email address is for aliasing (like tsc@nodejs.org) to roles in handling reports and escalation.
+  *  We can use groups.io
+* Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email
+* Repo: openjs/moderation (not provisioned)
+  * A form submission could open an issue in the repo, to allow folks to discuss in context
+## Confidentiality and record-keeping
+Everything reported and discussed is confidential unless explicitly stated. For any facts to be made public, the reportee’s explicit permission is needed. All reports should be recorded, together with the discussion of it. A private moderation repo would satisfy the need for recordkeeping.
+## Steps/Escalation
+1. A report is filed: This is the first step, mostly defined in Channels above.
+2. The right people are alerted: See Roles.
+3. Contact person assigned: See Roles.
+4. Report is acknowledged: The contact person responds to let the reportee know that the report is being discussed.
+5. Facts are gathered: Time is allocated to collect information in one place to make sure everyone involved has access.
```

Comment:
Are dissenting views and opinions "facts"? In other words, is there a possibility here to get opinionated input from other parties involved in the subject of the report? Or is purely factual input the only sort allowed at this step?

Reply:
I'm of the opinion that facts are the cornerstone of rendering sound decisions. But as humans are involved and CoCs can involve subjective experiences, I'd be +1 to adding explicit mention of opinions.